README.md

```diff
@@ -508,7 +510,7 @@ updating the instruction directly.  This approach also ensures that the code
 modification path is reliable, since ftrace has been doing it successfully for
 years.
 
-**Q Is kpatch compatible with \<insert kernel debugging subsystem here\>?**
+**Q. Is kpatch compatible with \<insert kernel debugging subsystem here\>?**
```

Comment:
Can you also correct the line starting with "**Q: Is it possible to register a function that..." a few lines down.  (ie, `s/:/.` )

Reply:
Will do.